invenio_jobs/services/services.py

```diff
@@ -292,3 +301,47 @@ def stop(self, identity, job_id, run_id, uow=None):
         uow.register(TaskRevokeOp(str(run.task_id)))
 
         return self.result_item(self, identity, run, links_tpl=self.links_item_tpl)
+
+
+class JobLogService(BaseService):
+    """Job log service."""
+
+    def search(self, identity, params):
+        """Search for app logs."""
+        self.require_permission(identity, "search")
+        search_after = params.pop("search_after", None)
```

Comment:
Does this param require some kind of validation and/or sanitation? It looks like that there is no Marshmallow schema for it.

Reply:
There is here: https://github.com/inveniosoftware/invenio-jobs/pull/73/files#diff-9b919a9bafafc0bad009e041746155486f2d9f1ff532a8a413809e0b17880befR135